images/runtime/php/template.Dockerfile

```diff
@@ -8,6 +8,8 @@ COPY src/startupscriptgenerator/src .
 ARG GIT_COMMIT=unspecified
 ARG BUILD_NUMBER=unspecified
 ARG RELEASE_TAG_NAME=unspecified
+ARG AI_CONNECTION_STRING
+ENV ORYX_AI_CONNECTION_STRING=${AI_CONNECTION_STRING}
```

Comment:
Any reason these two lines are defined up here when they're defined down below?

Reply:
Great catch! Removed the redundant ones.